Tests/Functional/ControllerResolverTest.php

```diff
@@ -67,4 +69,19 @@ public function testAnnotationControllerServiceExtendingClassicService()
 
         $this->assertEquals('hello', $client->getResponse()->getContent());
     }
+
+    /**
+     * @runInSeparateProcess
+     */
+    public function testInvokableControllerAsService()
+    {
+        if (version_compare(Kernel::VERSION_ID, '20600') < 0) {
```

Comment:
This should work here as this part of the code will be part of JMSDiExtraBundle now.